Makefile

```diff
@@ -38,11 +38,11 @@ test: ## Run tests
 .PHONY: lint
 lint: ## Run linter
 	@echo "==> Linting all packages..."
-	golangci-lint run $(SOURCE_FILES)
+	golangci-lint run $(SOURCE_FILES) --fix
```

Comment:
Sorry if I wasn't clear, leave this as it is as this runs in the ci and the ci won't be able to actually fix the changes, but we have a `check` command here that calls test and lint, instead of calling lint let's call fix there, that one is more for humans

Reply:
Here I mean here in the make file, search for `check `